Day3/src/index.js

```diff
@@ -57,10 +58,13 @@ const Heading3 = () => <h3>To learn indepth Reactjs</h3>;
 
 const TitleComponentWithComposition = () => {
   return (
-    <div className='title'>
-      <Heading1 />
-      <Heading2 />
-      <Heading3 />
+    <div className='page'>
+      <Navbar />
+      <main>
+        <Heading1 />
```

Comment:
Can you put the tags in one component? Are they being used in some other place?

Reply:
Fine, putting Headings into one separate component.